multicluster/controllers/multicluster/common/helper.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 Antrea Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "k8s.io/apimachinery/pkg/types"
+
+const (
+	AntreaMcsLabel        = "antrea.io/multi-cluster"
+	AntreaMcsAutoGenLabel = "antrea.io/multi-cluster-autogenerated"
+	SourceImportLabel     = "antrea.io/resource-import-name"
+
+	ServiceKind       = "Service"
+	EndpointsKind     = "Endpoints"
+	ServiceImportKind = "ServiceImport"
+
+	SourceServiceType = "SourceServiceType"
+
+	DefaultWorkerCount = 5
+
+	ResourceExportFinalizer = "resourceexport.finalizers.antrea.io"
+)
+
+const (
+	Separator = '/'
+)
+
+// TODO: Use NamespacedName stringer method instead of this. eg nsName.String()
+func NamespacedName(namespace, name string) string {
+	return namespace + string(Separator) + name
+}
+
+// GetNamespacedName returns the objects name as NamespacedName struct.
+func GetNamespacedName(namespace, name string) types.NamespacedName {
+	return types.NamespacedName{
```

Comment:
Do we need a func for this?

Reply:
yes, it help us to build up a new types.NamespacedName with given strings, not from K8s object directly, so we construct it by ourself, mainly purpose is to avoid duplicate codes, only remote_resourceimport_controller.go use it for now, let me move it from helper to remote_resourceimport_controller.go

Reply:
But types.NamespacedName{namespace, name} is very simple and just one line.

Reply:
yeah, we can use types.NamespacedName{namespace, name}  but it will give us a warning about `k8s.io/apimachinery/pkg/types.NamespacedName composite literal uses unkeyed fieldscomposites`, so I feel it's a little bit long to have both key and name.

Reply:
let me change it back to struct directly, so we have less function

Reply:
done